app/views/contexts/hub_context.py

```diff
@@ -49,19 +35,35 @@ class HubContext(Context):
     }
 
     def get_context(self, survey_complete, enabled_section_ids) -> Mapping:
-        context = self.HUB_CONTENT_STATES[
-            "COMPLETE" if survey_complete else "INCOMPLETE"
-        ]
-        context["rows"] = self._get_rows(enabled_section_ids)
-
-        return context
+        hub = self._schema.get_hub()
```

Comment:
Minor, but as hub could also refer to the hub context, it would be clearer if `hub` was named `hub_schema` here.